### ISSUE TITLE
Support for other chunk size

### ISSUE THREAD

**Issue by oschijns** [Inspirateur/binary-greedy-meshing#8]
As I intend to use this library with lower-end hardware, it would be useful to provide support for chunk-size other than 64³ voxels. From what I understand face visibility is evaluated through bitwise operation with one bit padding around the chunk (62³ voxels).

I guess 8³, 16³ and 32³ could be supported through the use of feature flags or by using templates ?

**Comment by Inspirateur** [comment]
Yes it is exactly as you say. We can make it work with smaller bit sizes, I'm going to look into templating :)

**Comment by Inspirateur** [comment]
Sorry for being so slow on this issue, i've been busy with other projects. But I've been thinking: wouldn't using "usize" instead of "u64" solve your usecase ?

**Comment by oschijns** [comment]
Hello, no problem.
I am myself really busy with personal stuff.

I think it is worth keeping this issue open, if either one of us has the time to implement the improvement.

But there is no imminent need.
